Modules/Devices/src/Devices.Application/Infrastructure/PushNotifications/DeletionProcess/DeletionProcessWaitingForApprovalReminderPushNotification.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Backbone.Modules.Devices.Application.Infrastructure.PushNotifications.DeletionProcess;
+
+[NotificationText(Title = "Deletion process waiting for approval.",
+    Body = "There is a deletion process for your identity that waits for your approval. If you don't approve it within a few days, it will be closed.")]
```

Comment:
`closed` is such an ambiguous word in this context. Can't we come up with something a bit more intuitive?